test/unit/gateways/paymill_test.rb

```diff
@@ -181,6 +181,32 @@ def test_successful_store
     assert response.test?
   end
 
+  def test_store_includes_currency
+    expected_currency = 'USD'
+
+    @gateway.expects(:raw_ssl_request).with(
+      :get,
```

Comment:
What does this line do?

Reply:
```store_endpoint``` is just a method that returns a string. That string is the second expected argument in the ```raw_ssl_request``` call.

Reply:
Oooh, I think I see now. So the `.with()`method requires _all_ of the parameters that are passed to the given function, meaning that you needed to pass in everything here:
```
def raw_ssl_request(method, endpoint, data, headers = {})
```
https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/posts_data.rb#L47

That's where I had been confused. I didn't know what the four different parameters listed out here were doing.